IBPSA/Fluid/Movers/FlowControlled_m_flow.mo

```diff
@@ -106,6 +117,13 @@ in record <code>per</code>, which is of type
 IBPSA.Fluid.Movers.SpeedControlled_Nrpm</a>.
 </p>
 <p>
+Note that if the model operates with a head that is larger than <code>dpMax</code>, which by default is
+two times larger than the largest head declared in <code>eff.per.pressure.dp</code>,
+the simulation will stop with an error message.
+This guards against unreasonably high pressure drops which can happen if the pump or fan is forcing mass flow rate
```

Comment:
```suggestion
This guards against unreasonably high pressure drops and electrical power uses, which can happen if the pump or fan is forcing mass flow rate
```

Reply:
I will commit it, and remove the 's' in "uses" -- here there is no trailing 's'.